monitoring-guidelines.md

```diff
@@ -55,13 +55,21 @@ When creating a KubeVirt alert rule, please follow the [OpenShift Alerting Consi
 In addition to the [OpeShift Style Guide](https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/alerting-consistency.md#style-guide) the KubeVirt alerts MUST include:
 1. `kubernetes_operator_part_of` label indicating the operator name. Value should be set to `kubevirt`.
 2. `kubernetes_operator_component` label indicating the value of the sub operator name.
-
+3. `operator_health_impact` label indicating how the alert impacts the operator's functionality.
+   This label differs from `severity`, as `severity` can be tied to workload issues (e.g VMIs), while  `operator_health_impact` refers only to the impact the issue has on the operator itself.
+   For example, An alert may have a `warning` severity, when talking on the impact on the cluster health, but have a `critical` impact on the operator health. 
+   Also, When an alert is tied to a specific workload it can have a `warning` severity, but no impact on the operator health.
```

Comment:
@assafad Please note that the difference between the severity and this label is not only because of the alert being tied to a workload.  It can be because it doesn't affect the cluster level but it does affect on the operator health level.

This label differs from `severity`, as the `severity` indicates the ability to deliver a service **for the cluster as a whole**. Where the `operator_health_impact` indicates the impact of the issue **on the operator functionality**.
The loss of the operator functionality doesn't necessarily mean that the the ability to deliver services for the cluster as a whole is affected.

For example, An alert may have a "warning" severity, when talking on the impact on the cluster health, but have a "critical" impact on the operator health. Also, When an alert is tied to a specific workload it can have a "warning" severity, but no impact on the operator health.

Reply:
Updated